.github/workflows/release.yml

```diff
@@ -5,13 +5,17 @@ on:
     types: [prereleased, released]
 
 jobs:
-  release-s3:
-    name: Release tarball & config
+  release-packages:
+    name: Release tarball
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goarch: [ amd64, arm64, arm ]
```

Comment:
in the case of arm , is it v6 by default ? i don't remeber if that was goreleaser or go doing the default

Reply:
yes it is v6 
```
$GOARM (for arm only; default is auto-detected if building on the target processor, 6 if not)
If in doubt, leave this variable unset, and adjust it if required when you first run the Go executable. The GoARM page on the Go community wiki contains further details regarding Go's ARM support.
```

https://golang.org/doc/install/source